docs/source/index.rst

```diff
@@ -1,6 +1,10 @@
 Welcome to bats-core's documentation!
 =====================================
 
+Bats (Bash Automated Testing System) is a `TAP <https://testanything.org/>`_-compliant testing framework for Bash 3.2 or above. It provides a simple way to verify that the UNIX programs you write behave as expected.
+
+Bats is most useful when testing software written in Bash, but you can use it to test any UNIX program.
+
 Versions before v1.2.1 are documented over `there <https://github.com/bats-core/bats-core/blob/master/docs/versions.md>`_.
```

Comment:
Thanks for adding this! If you can include this tiny tweak (unrelated to your change), that would be a great bonus.

```suggestion
Versions before v1.2.1 are documented over in `docs/versions.md <https://github.com/bats-core/bats-core/blob/master/docs/versions.md>`_.
```